src/include/daos_task.h

```diff
@@ -142,9 +142,6 @@ typedef struct {
 } daos_set_params_t;
 
 typedef struct {
-	uint32_t		mode;
```

Comment:
Those structs should always mirror what the actual API looks like, so you can't remove mode/uid/gid if those remain in the API. Those are ways where we can call the DAOS API in a different manner than the daos_xyz() functions.

Reply:
Fixed